### DISCUSSION TITLE
Comparing Negative values in DeepAR Loss

### DISCUSSION THREAD

**Discussion by anandmohan4143** [awslabs/gluonts#1534] [Q&A] [answered]
Hi @lostella , I am a student and is working on gluonts deepAR with gluonts version=0.5.0. I am getting negative value like -0.4 as the Final Loss after DeepAR model training. Also tried out another DeepAR model and got final loss as -0.7. How to compare and decide which model is better?  am not sure how to compare two negative log likelihood - negative values. I google it out but no clear description is given. Kindly help me.

**Comment by yitao-yu** [comment]
Probability is a value between (0,1). So taking a log, the log-likelihood is between -inf and 0. The closer to 0 it is, it's better. 

If you are comparing models, you probably should have a separated validation dataset, and compute the loss there. The validation set should not be used for training, but just for comparing models.

**Reply by lostella** [reply]
Note that probability densities are not upper bounded by 1 in general (unlike probability masses, in the case of discrete distribution), so the log density can range anywhere between -inf and +inf

**Reply by yitao-yu** [reply]
True, my bad.

**Comment by mbohlkeschneider** [comment, marked as answer]
The lower the loss the better. Negative loss values can happen and are no reason for concern.
